### PR TITLE
MusicAlbum.sql

### LINKED ISSUE #27: [1pt] Create a schema.sql for the Author and game class
- Create a schema.sql file with tables that will be analogical to the structure of the classes that you created:
    - games table (add all properties and associations from the parent Item class as table columns)
    - authors table

### LINKED ISSUE #42: [1pt] Create a schema.sql file with tables that will be analogical to the structure of the classes that you created: **labels table**
Provide tables that will be analogical to the structure of the classes that you created: 
 - labels table (add all properties and associations from the parent Item class as table columns)

### LINKED ISSUE #60: [ 1pts ] Create a schema.sql file
Provide tables that will be analogical to the structure of the classes that you created: 
 - music_albums table (add all properties and associations from the parent Item class as table columns)
 - genres table

### PR DESCRIPTION
**Pull request by AmaliaGM**
# Provide tables that will be analogical to the structure of the classes that you created:
 
music_albums table (add all properties and associations from the parent Item class as table columns)
genres table
- close #60
- close #27 
- close #42

### CODE REVIEW

File: MusicAlbum.sql [hunk 1]
```diff
@@ -0,0 +1,22 @@
+-- Table: public.MusicAlbum
+
+-- DROP TABLE IF EXISTS public."MusicAlbum";
+
+CREATE TABLE IF NOT EXISTS public."MusicAlbum"
+(
+    id integer NOT NULL GENERATED ALWAYS AS IDENTITY ( INCREMENT 1 START 1 MINVALUE 1 MAXVALUE 2147483647 CACHE 1 ),
+    "Label" text COLLATE pg_catalog."default" NOT NULL,
```
Comment (andym80) [inline MusicAlbum.sql:8]: Please, quit **" " ** from Label

File: MusicAlbum.sql [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    genre text COLLATE pg_catalog."default" NOT NULL,
+    author text COLLATE pg_catalog."default" NOT NULL,
+    source text COLLATE pg_catalog."default",
+    archived boolean,
+    publish_date date,
+    on_spotify boolean,
+    "can_be_archived?()" boolean
```
Comment (andym80) [inline MusicAlbum.sql:15]: Please, quit **" " ** from   can_be_archived?()

**Review by andym80 — CHANGES_REQUESTED** [review]

File: MusicAlbum.sql (hunk 2 above)
Comment (carlosmuerto) [inline MusicAlbum.sql:15]: can_be_archived?() is a function of the object, in the UML class diagram and does not bellow to the database.

**Review by carlosmuerto — CHANGES_REQUESTED** [review]
Please move all .sql files to a database folder, to keep the root folder clear.

**Review by andym80 — APPROVED** [review]

**Review by carlosmuerto — APPROVED** [review]

**Pull request merged**